google/cloud/storage/bucket.py

```diff
@@ -617,12 +617,13 @@ def blob(
 
     def notification(
         self,
-        topic_name,
+        topic_name=None,
         topic_project=None,
         custom_attributes=None,
         event_types=None,
         blob_name_prefix=None,
         payload_format=NONE_PAYLOAD_FORMAT,
+        notification_id=None,
```

Comment:
I'd call this arg just `id_`. As method straightly says that you're getting notification, there is no need in such a long name

Reply:
Yes i used  id first , but 'id' is a built in name in python and even in library they used a long name rather than id so i used it.

Reply:
Standards says, if your variable name crosses builtin, you can add `_` to an end of your variable name

Reply:
Yes but they never used in any library so i an unsure about.